release notes/v0.36.0.md

```diff
@@ -0,0 +1,76 @@
+
+Following [#2279](https://github.com/grafana/k6/pull/2279), the `k6 inspect` command now supports an `--execution-requirements` flag. When used, the command's output will include fields related to the execution requirements, the configuration K6 derived from the execution context, and will also expose the `maxVUs` and `totalDuration` fields.
+
+
+### Forcing HTTP/1 protocol ([#2222](https://github.com/grafana/k6/pull/2222))
+
+Thanks to the work of [@sjordhani22](https://github.com/sjordhani22), [#2222](https://github.com/grafana/k6/pull/2222) made it possible to force K6 to use version 1.1 of the protocol when firing HTTP requests. 
+
+It can be done by setting the `http2client=0` value in the `GODEBUG` environment variable:
+
+```
+GODEBUG=http2client=0 k6 run testscript.js
+```
+
+**Nota Bene**: the usage of the `GODEBUG` variable is considered temporary, and expected to change in the future. If you start using this feature, keep an eye out for potential future changes.
+
+## Bugs Fixed!
+
+* We had a long-standing issue ([#1771](https://github.com/grafana/k6/issues/1771)) making opening large files in the context of a K6 script somewhat unreliable under specific conditions in the past. This has now been addressed by [#2314](https://github.com/grafana/k6/pull/2314).
+* We addressed an issue uncovered by [our community](https://community.k6.io/t/v0-35-0-grpc-server-reflection-error/2383), which kept our users using GRPC to define multiple services in a single *proto* file. This issue was addressed by [#2265](https://github.com/grafana/k6/pull/2265).
+* Thanks to the contribution of [@Resousse](https://github.com/Resousse), we've now updated K6's `go-ntlmssp` dependency. The updating PR [#2290](https://github.com/grafana/k6/pull/2290) indeed fixes issues with NTLM Authentication backends returning two authorization headers. 
+
+## Maintenance
+
+- `v0.36.0` marks the switch to a new JS module's internal API. All our official JS modules have now been ported to this new version, which should make the process of developing modules easier and more streamlined in the future. See for more details:  [#2243](https://github.com/grafana/k6/pull/2243), [#2241](https://github.com/grafana/k6/pull/2241), [#2239](https://github.com/grafana/k6/pull/2239), [#2242](https://github.com/grafana/k6/pull/2242), [#2226](https://github.com/grafana/k6/pull/2226), and [#2232](https://github.com/grafana/k6/pull/2232).
+- We have refactored our implementation of the RampingVU executor, for better clarity and maintainability. See [#2155](https://github.com/grafana/k6/pull/2155).
+- [#2316](https://github.com/grafana/k6/pull/2316) relaxed quite a few of the code linting rules we applied to K6's code.
+- [#2304](https://github.com/grafana/k6/pull/2304) removed external dependencies from K6's JSONAPI compliant Rest API: allowing us to both reduce our reliance on external dependencies and improve its maintainability.
```

Comment:
```suggestion
- [#2304](https://github.com/grafana/k6/pull/2304) ~~removed~~ external dependencies from K6's JSONAPI compliant Rest API: allowing us to both reduce our reliance on external dependencies and improve its maintainability.
```
IIRC, we didn't really remove the dependency, we just deprecated some methods and we will remove when https://github.com/grafana/k6/pull/2313 will be merged.

Reply:
My bad then, I thought I remembered we proceeded with actually removing it. Thanks for the heads up 🙇🏻

Reply:
Yep, it's going to be removed in the next release.

This version deprecates the `(api.v1)client.Call` method. Instead, there is a new version `client.CallAPI`.

Reply:
I updated the notes accordingly, thanks a lot @olegbespalov that was super helpful 🙇🏻